docker-compose.ci.yml

```diff
@@ -22,46 +22,42 @@ services:
       REDIS_HOST: redis
       MONGO_HOST: mongo
       POSTGRES_HOST: postgres
-      ENABLE_CONVERSIONS: "true"
       MOCHA_GREP: ${MOCHA_GREP}
       NODE_ENV: test
+      ENABLE_CONVERSIONS: "true"
       USE_PROM_METRICS: "true"
-      AWS_KEY: fake
-      AWS_SECRET: fake
       AWS_S3_USER_FILES_BUCKET_NAME: fake_user_files
       AWS_S3_TEMPLATE_FILES_BUCKET_NAME: fake_template_files
       AWS_S3_PUBLIC_FILES_BUCKET_NAME: fake_public_files
-      AWS_S3_ENDPOINT: http://fakes3:9090
+      AWS_S3_ENDPOINT: http://s3:9090
+      AWS_ACCESS_KEY_ID: fake
+      AWS_SECRET_ACCESS_KEY: fake
     depends_on:
       mongo:
         condition: service_healthy
       redis:
         condition: service_healthy
-      fakes3:
+      s3:
         condition: service_healthy
     user: node
     command: npm run test:acceptance:_run
 
 
-
   tar:
     build: .
     image: ci/$PROJECT_NAME:$BRANCH_NAME-$BUILD_NUMBER
     volumes:
       - ./:/tmp/build/
     command: tar -czf /tmp/build/build.tar.gz --exclude=build.tar.gz --exclude-vcs .
     user: root
-
   redis:
     image: redis
 
   mongo:
     image: mongo:3.4
-
-  fakes3:
+  s3:
     image: adobe/s3mock
     environment:
       - initialBuckets=fake_user_files,fake_template_files,fake_public_files
-    healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:9090"]
-
+  healthcheck:
```

Comment:
This looks like a bad indentation?

Reply:
Thanks. :+1: auto-indentation error in the source jinja files. :/